### ISSUE TITLE
Not working with the new Firestore (former Firebase DB)

### ISSUE THREAD

**Issue by concernedrat** [sidferreira/aor-firebase-client#31]
Any plans to support the new React Admin and Firestore?

**Comment by bvivek20** [comment]
Guys, any plans of supporting firestore?

**Comment by jesster2k10** [comment]
@bvivek20 @georgerb I would say it wouldn't be that hard to do. I'm going to attempt it myself, what you should look at is the `moments.js` file as that's where the Old DB code is.

**Comment by georgerb** [comment]
yeah @jesster2k10 I did it but was hacky AF. Firestore has new features and the pagination is a little bit annoying which makes the porting a little bit harder.

**Comment by sidferreira** [comment]
@georgerb @bvivek20 @jesster2k10 I'm trying to setup some time to work on it in the coming days. Sorry the delay so far.

**Comment by georgerb** [comment]
@sidferreira yo! you rock! thanks for sparing a little bit of time on this!

**Comment by concernedrat** [comment]
Hi. Any updates?
